docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-examples/managed-cli-example-create.md

```diff
@@ -213,6 +213,201 @@ Name              Tier        Version        State     Health    Regions
+  --credentials username=admin,password=password \
+  --fault-tolerance=ZONE --region-info region=eastus,num-nodes=3,num-cores=4 \
+```
+
+```output
+The cluster my-sandbox has been created
+Name           Tier        Version           State     Health    Provider   Regions     Nodes     Node Res.(Vcpu/Mem/DiskGB/IOPS)
+my-sandbox   Dedicated   {{< yb-version version="preview" format="build">}}       ACTIVE    💚        AZURE      eastus   3         4 / 16GB / 200GB / 3000
+```
+
+You can list the EAR details using the encryption list command.
+
+```sh
+ybm cluster encryption list --cluster-name my-sandbox
+```
+
+```output
+Provider   Key Alias                              Last Rotated               Security Principals                      CMK Status
+AZURE      8aXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX5b   2023-11-03T07:37:26.351Z   <your-key-vault-uri>   ACTIVE
+```
+
+EAR details are also shown when you use `cluster describe` command.
+
+{{% /tab %}}
+
+{{< /tabpane >}}
+
+### Rotate your CMK
+
+Use the following commands to rotate your CMK. You can also use these commands to encrypt a cluster where the specified cluster does not already have EAR.
+
+Note: Only credentials can be modified in the current configuration (for example, AWS access/secret keys or GCP service account credentials).
+
+{{< tabpane text=true >}}
+
+{{% tab header="AWS" %}}
+
+```sh
+ybm cluster encryption update \
+  --cluster-name my-sandbox \
+  --encryption-spec cloud-provider=AWS,aws-secret-key=<new-secret-key>,aws-access-key=<new-access-key>
+```
+
+{{% /tab %}}
+
+{{% tab header="GCP" %}}
+
+```sh
+ybm cluster encryption update \
+  --cluster-name my-sandbox \
+  --encryption-spec cloud-provider=GCP,gcp-resource-id=projects/yugabyte/locations/global/keyRings/test-byok/cryptoKeys/key1,gcp-service-account-path=<path-to-service-account-file> 
+```
+
+{{% /tab %}}
+
+{{% tab header="Azure" %}}
+
+```sh
+ybm cluster encryption update \
+  --cluster-name my-sandbox \
+  --encryption-spec cloud-provider=AZURE,azu-client-id=<new-client-id>,azu-client-secret=<new-client-secret>,azu-tenant-id=<new-tenant-id>,azu-key-name=test-key,azu-key-vault-uri=<new-key-vault-uri>
+```
+
+{{% /tab %}}
+
+{{< /tabpane >}}
+
+### Enable and disable EAR
+
+To disable EAR on a cluster, use the following command:
+
+```sh
+ybm cluster encryption update-state \
+  --cluster-name my-sandbox \
+  --disable
+```
+
+```output
+Successfully DISABLED encryption at rest for cluster my-sandbox
+```
+
+After you disable EAR, YugabyteDB Aeon uses lazy decryption to decrypt the cluster.
+
+You can check the status of the EAR using the encryption list command.
+
+```sh
+ybm cluster encryption list --cluster-name my-sandbox
+```
+
+```output
+Provider   Key Alias                              Last Rotated               Security Principals                      CMK Status
+AZURE      8aXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX5b   2023-11-03T07:37:26.351Z   <your-key-vault-uri>   NOT_ACTIVE
+```
```

Comment:
@arishta-yb I didn't get any output for this command when CMK is disabled. Could you tell me what the CMK status would be in that case? Would it be NOT_ACTIVE or DISABLED in the `encryption list` command output?